Driver Code/SchemaBuilder.cs

```diff
@@ -144,15 +144,13 @@ private void VerifyEntityAssemblyVersion()
 			{
 				throw new InvalidOperationException("No ILinqMetaData type found.");
 			}
-			var ormSupportClassesAssemblyName = _linqMetaDataType.Assembly.GetReferencedAssemblies()
-													.Where(an=>(an.Name=="SD.LLBLGen.Pro.ORMSupportClasses") && 
-																(an.Version.Major==Constants.MajorVersion) &&
-																(an.Version.Minor==Constants.MinorVersion)).FirstOrDefault();
-			if(ormSupportClassesAssemblyName == null)
-			{
-				throw new InvalidOperationException(string.Format("The assembly '{0}' is not compiled against the LLBLGen Pro Runtime Framework v{1}.{2}.",
-					CxInfoHelper.GetEntityAssemblyFilename(_cxInfo, CxInfoHelper.GetTemplateGroup(_cxInfo)), Constants.MajorVersion, Constants.MinorVersion));
-			}
+
+
+			var ormSupportClassesAssemblyName = _linqMetaDataType.Assembly.GetReferencedAssemblies().FirstOrDefault(an => an.Name=="SD.LLBLGen.Pro.ORMSupportClasses");
+            if(ormSupportClassesAssemblyName == null) throw new InvalidOperationException($"The assembly '{CxInfoHelper.GetEntityAssemblyFilename(_cxInfo, CxInfoHelper.GetTemplateGroup(_cxInfo))}' is not found.");
```

Comment:
Why this change, if I may ask? (And spaces over tabs... tssk! ;) )

Reply:
I couldn't get the driver to work in Linq pad and got the error _"The assembly '{0}' is not compiled against the LLBLGen Pro Runtime Framework v5.5."_ I wanted to know whether it couldn't find the dll or it was a version problem. Sorry about the spaces :( my bad

Reply:
Hmm, so our code is buggy... will check it out

Reply:
It wasn't a bug. It turned out VS used an old version of the ORMSupport classes. I got this error in Linqpad but didn't understand why. So I made this change so I could see whether the DLL even got loaded and if so which version. In my case it got loaded and the error showed it got the wrong version. It helped me pinpoint the problem easier. That's all. I thought it might help someone else as well.